license files (#2625)
SoftwareAG/c8y-docs: content/legal-notices/Acceptable-Use-Policy.md

> ### General Conduct

Reviewer: ```suggestion
### General conduct
```

> Customer must use the Cumulocity GmbH Services and Infrastructure in a manner consistent with the permitted use of such Cumulocity GmbH Services and Infrastructure. Unless otherwise expressly permitted in writing by Cumulocity GmbH, Customer may not assign, transfer, distribute, resell, lease or otherwise provide access to any third party to the Cumulocity GmbH Services and Infrastructure, or use the Cumulocity GmbH Services and Infrastructure with or for hte benefit of any third party (other than Internet end users). Customer may only use the Cumulocity GmbH Services and Infrastructure for lawful purposes and in accordance with this AUP.

Reviewer: ```suggestion
Customer must use the Cumulocity GmbH Services and Infrastructure in a manner consistent with the permitted use of such Cumulocity GmbH Services and Infrastructure. Unless otherwise expressly permitted in writing by Cumulocity GmbH, Customer may not assign, transfer, distribute, resell, lease or otherwise provide access to any third party to the Cumulocity GmbH Services and Infrastructure, or use the Cumulocity GmbH Services and Infrastructure with or for the benefit of any third party (other than Internet end users). Customer may only use the Cumulocity GmbH Services and Infrastructure for lawful purposes and in accordance with this AUP.
```

> Customer must use the Cumulocity GmbH Services and Infrastructure in a manner consistent with the permitted use of such Cumulocity GmbH Services and Infrastructure.  Unless otherwise expressly permitted in writing by Cumulocity GmbH, Customer may not assign, transfer, distribute, resell, lease or otherwise provide access to any third party to the Cumulocity GmbH Services and Infrastructure, or use the Cumulocity GmbH Services and Infrastructure with or for hte benefit of any third party (other than Internet end users). Customer may only use the Cumulocity GmbH Services and Infrastructure for lawful purposes and in accordance with this AUP.

Reviewer: ```suggestion
Customer must use the Cumulocity GmbH Services and Infrastructure in a manner consistent with the permitted use of such Cumulocity GmbH Services and Infrastructure. Unless otherwise expressly permitted in writing by Cumulocity GmbH, Customer may not assign, transfer, distribute, resell, lease or otherwise provide access to any third party to the Cumulocity GmbH Services and Infrastructure, or use the Cumulocity GmbH Services and Infrastructure with or for hte benefit of any third party (other than Internet end users). Customer may only use the Cumulocity GmbH Services and Infrastructure for lawful purposes and in accordance with this AUP.
```

> ### Responsibility for Content

Reviewer: ```suggestion
### Responsibility for content
```

> Cumulocity GmbH takes no responsibility for any Customer or User content created, accessible or delivered on or through the Cumulocity GmbH Services and Infrastructure.  Cumulocity GmbH does not monitor or exercise any editorial control over such content. Customer is solely responsible for (i) any content published or made available through the Cumulocity GmbH Services and Infrastructure by Customer and its Users and (ii) compliance with all laws applicable to the publication and distribution of such content.

Reviewer: ```suggestion
Cumulocity GmbH takes no responsibility for any Customer or User content created, accessible or delivered on or through the Cumulocity GmbH Services and Infrastructure. Cumulocity GmbH does not monitor or exercise any editorial control over such content. Customer is solely responsible for (i) any content published or made available through the Cumulocity GmbH Services and Infrastructure by Customer and its Users and (ii) compliance with all laws applicable to the publication and distribution of such content.
```

> ### Inappropriate and Illegal Content

Reviewer: ```suggestion
### Inappropriate and illegal content
```

> Customer shall not use the Cumulocity GmbH Services and Infrastructure to transmit, distribute or store material that is inappropriate (including online gambling), as reasonably determined by Cumulocity GmbH, or material that is illegal, defamatory, libelous, indecent, obscene, pornographic, enables online gambling or inconsistent with the generally accepted practises of the internet community. Customer shall ensure that its and its Users’ use of the Cumulocity GmbH Services and Infrastructure and all content transmitted, distributed or stored on the Cumulocity GmbH Infrastructure does not violate any applicable domestic or foreign laws or regulations including but not limited to laws relating to content distribution, encryption or export or any rights of any third party.  Customer shall not use the Cumulocity GmbH Services and Infrastructure to transmit, distribute or store material that contains a virus, worm, Trojan horse or other component harmful to the Cumulocity Services and Infrastructure, any other infrastructure or equipment, or other Users.

Reviewer: ```suggestion
Customer shall not use the Cumulocity GmbH Services and Infrastructure to transmit, distribute or store material that is inappropriate (including online gambling), as reasonably determined by Cumulocity GmbH, or material that is illegal, defamatory, libelous, indecent, obscene, pornographic, enables online gambling or inconsistent with the generally accepted practises of the internet community. Customer shall ensure that its and its Users’ use of the Cumulocity GmbH Services and Infrastructure and all content transmitted, distributed or stored on the Cumulocity GmbH Infrastructure does not violate any applicable domestic or foreign laws or regulations including but not limited to laws relating to content distribution, encryption or export or any rights of any third party. Customer shall not use the Cumulocity GmbH Services and Infrastructure to transmit, distribute or store material that contains a virus, worm, Trojan horse or other component harmful to the Cumulocity Services and Infrastructure, any other infrastructure or equipment, or other Users.
```

> ### Intellectual Property

Reviewer: ```suggestion
### Intellectual property
```

> Customer shall not use the Cumulocity GmbH Services and Infrastructure in any manner that would infringe, dilute, misappropriate or otherwise violate any privacy or other personal rights or any intellectual property rights, including but not limited to, copyrights and laws protecting patents, trademarks, trade secrets or other proprietary information.  If Customer uses a domain name in connection with its use of the Cumulocity GmbH Services and Infrastructure, such domain name must not violate any trademark, service mark, or other rights of any third party.

Reviewer: ```suggestion
Customer shall not use the Cumulocity GmbH Services and Infrastructure in any manner that would infringe, dilute, misappropriate or otherwise violate any privacy or other personal rights or any intellectual property rights, including but not limited to, copyrights and laws protecting patents, trademarks, trade secrets or other proprietary information. If Customer uses a domain name in connection with its use of the Cumulocity GmbH Services and Infrastructure, such domain name must not violate any trademark, service mark, or other rights of any third party.
```

> ### Fraudulent/Misleading Content

Reviewer: ```suggestion
### Fraudulent/misleading content
```

Customer shall not use the Cumulocity GmbH Services and Infrastructure to transmit or distribute material containing fraudulent offers for goods or services, or any advertising or promotional materials that contain false, deceptive, or misleading statements, claims or representations.

> ### Email and Spam

Reviewer: ```suggestion
### Email and spam
```

> Customer shall not use the Cumulocity GmbH Services and Infrastructure to send unsolicited email messages, including without limitation, bulk commercial advertising or informational announcements (“spam”).  Further, Customer is prohibited from using the service of another provider to send spam or to otherwise promote a site hosted on or connected to the Cumulocity GmbH Infrastructure.  In addition, Customer shall not use the Cumulocity GmbH Services and Infrastructure to (a) send email messages which are excessive and/or intended to harass or annoy others, (b) continue to send email messages to a recipient who has indicated that he/she does not wish to receive them, © send email messages with forged header information, or (d) send malicious email messages, including, without limitation, “mailbombing”.  Cumulocity GmbH reserves the right to charge Customer at Cumulocity GmbH’s standard rates for time required to handle any complaints that Customer or User violate this Email and Spam section.  For the purpose of this Policy and in the event of doubt, the legislation of the recipient shall apply for any classification of the above prohibited usages.

Reviewer: ```suggestion
Customer shall not use the Cumulocity GmbH Services and Infrastructure to send unsolicited email messages, including without limitation, bulk commercial advertising or informational announcements (“spam”). Further, Customer is prohibited from using the service of another provider to send spam or to otherwise promote a site hosted on or connected to the Cumulocity GmbH Infrastructure. In addition, Customer shall not use the Cumulocity GmbH Services and Infrastructure to (a) send email messages which are excessive and/or intended to harass or annoy others, (b) continue to send email messages to a recipient who has indicated that he/she does not wish to receive them, © send email messages with forged header information, or (d) send malicious email messages, including, without limitation, “mailbombing”.  Cumulocity GmbH reserves the right to charge Customer at Cumulocity GmbH’s standard rates for time required to handle any complaints that Customer or User violate this Email and Spam section.  For the purpose of this Policy and in the event of doubt, the legislation of the recipient shall apply for any classification of the above prohibited usages.
```

> ### Security Violations

Reviewer: ```suggestion
### Security violations
```

> ### Cumulocity GmbH Rights and Remedies

Reviewer: ```suggestion
### Cumulocity GmbH rights and remedies
```

> If Customer becomes aware of any content or activity that violates this AUP, Customer shall take all necessary action to prevent such Content from being routed to, passed through, or stored on the Cumulocity GmbH Infrastructure.  To the extent Cumulocity GmbH becomes aware of any content or activities that Cumulocity GmbH deems, in its sole discretion, to be in violation of this AUP, Cumulocity GmbH may immediately block access to such content, suspend or terminate any affected Cumulocity GmbH Live Services, or take any other actions Cumulocity GmbH deems appropriate.  Cumulocity GmbH shall be solely responsible for determining if any content or action violates this AUP.  Cumulocity GmbH reserves the right to terminate or suspend services if the continued provision of services would violate law or otherwise harm Cumulocity GmbH’s Infrastructure, Services or customers.  Cumulocity GmbH also reserves the right to cooperate with legal authorities and third parties in the investigation of alleged wrongdoing, including disclosing the identity of the party that Cumulocity GmbH deems responsible for the wrongdoing.  Cumulocity GmbH will endeavor to provide notice to Customer prior to suspension or termination of Cumulocity GmbH Live Services but may immediately suspend or terminate in instances where continued provision of Cumulocity GmbH Live Services would have a material adverse effect on Cumulocity GmbH.  Cumulocity GmbH shall not be liable for any damages of any nature suffered by any Customer, User, or any third party resulting in whole or in part from Cumulocity GmbH’s exercise of its rights under the AUP.

Reviewer: ```suggestion
If Customer becomes aware of any content or activity that violates this AUP, Customer shall take all necessary action to prevent such content from being routed to, passed through, or stored on the Cumulocity GmbH Infrastructure. To the extent Cumulocity GmbH becomes aware of any content or activities that Cumulocity GmbH deems, in its sole discretion, to be in violation of this AUP, Cumulocity GmbH may immediately block access to such content, suspend or terminate any affected Cumulocity GmbH Live Services, or take any other actions Cumulocity GmbH deems appropriate. Cumulocity GmbH shall be solely responsible for determining if any content or action violates this AUP. Cumulocity GmbH reserves the right to terminate or suspend services if the continued provision of services would violate law or otherwise harm Cumulocity GmbH’s Infrastructure, Services or customers. Cumulocity GmbH also reserves the right to cooperate with legal authorities and third parties in the investigation of alleged wrongdoing, including disclosing the identity of the party that Cumulocity GmbH deems responsible for the wrongdoing. Cumulocity GmbH will endeavor to provide notice to Customer prior to suspension or termination of Cumulocity GmbH Live Services but may immediately suspend or terminate in instances where continued provision of Cumulocity GmbH Live Services would have a material adverse effect on Cumulocity GmbH. Cumulocity GmbH shall not be liable for any damages of any nature suffered by any Customer, User, or any third party resulting in whole or in part from Cumulocity GmbH’s exercise of its rights under the AUP.
```

> Cumulocity GmbH reserves the right to modify this AUP from time to time, in its sole discretion, effective upon posting a revised copy of the Acceptable User Policy on *Link to the policy*. Any use of the Cumulocity GmbH Services and Infrastructure after such modification shall constitute acceptance of such modification.

Reviewer: ```suggestion
Cumulocity GmbH reserves the right to modify this AUP from time to time, in its sole discretion, effective upon publishing a revised version of the Acceptable User Policy. Any use of the Cumulocity GmbH Services and Infrastructure after such modification shall constitute acceptance of such modification.
```

Reviewer: @JanePorter @pink-mongoose Please check once more and approve.

> Cumulocity GmbH reserves the right to modify this AUP from time to time, in its sole discretion, effective upon posting a revised copy of the Acceptable User Policy on *Link to the policy*. Any use of the Cumulocity GmbH Services and Infrastructure after such modification shall constitute acceptance of such modification.

Reviewer: @JanePorter Link to be added

Reviewer 2: Hi Beate, I believe this would be a circular link so I think we just remove that section and and the sentence after the word Policy. I thought I had removed this.

> Cumulocity GmbH reserves the right to modify this AUP from time to time, in its sole discretion, effective upon posting a revised copy of the Acceptable User Policy on *Link to the policy* .  Any use of the Cumulocity GmbH Services and Infrastructure after such modification shall constitute acceptance of such modification.

Reviewer: ```suggestion
Cumulocity GmbH reserves the right to modify this AUP from time to time, in its sole discretion, effective upon posting a revised copy of the Acceptable User Policy on *Link to the policy*. Any use of the Cumulocity GmbH Services and Infrastructure after such modification shall constitute acceptance of such modification.
```

> Cumulocity GmbH reserves the right to modify this AUP from time to time, in its sole discretion, effective upon posting a revised copy of the Acceptable User Policy on *Link to the policy* .  Any use of the Cumulocity GmbH Services and Infrastructure after such modification shall constitute acceptance of such modification.

Reviewer: @JanePorter Not sure which link must be added here.

### General conduct

> Customer must use the Cumulocity GmbH Services and Infrastructure in a manner consistent with the permitted use of such Cumulocity GmbH Services and Infrastructure. Unless otherwise expressly permitted in writing by Cumulocity GmbH, Customer may not assign, transfer, distribute, resell, lease or otherwise provide access to any third party to the Cumulocity GmbH Services and Infrastructure, or use the Cumulocity GmbH Services and Infrastructure with or for hte benefit of any third party (other than Internet end users). Customer may only use the Cumulocity GmbH Services and Infrastructure for lawful purposes and in accordance with this AUP.

Author: typo that I missed end of 2nd line hte --> the
